data/json/items/magazines.json

```diff
@@ -0,0 +1,78 @@
+[
+    {   "type" : "MAGAZINE",
+        "id" : "stanag30",
+        "price" : 8000,
+        "name" : "STANAG magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard capacity magazine for use with STANAG compatible rifles.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 110,
+        "to_hit" : -2,
+        "ammo_type" : "223",
+        "capacity" : 30,
+        "reliability" : 10,
+        "reload_time" : 30,
+        "flags" : [ "COMPACT_MAG" ]
+    },
+    {   "type" : "MAGAZINE",
+        "id" : "belt223",
+        "price" : 0,
+        "name" : ".223 ammo belt",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "An ammo belt consisting of metal linkages which disintegrate upon firing.",
+        "material" : "steel",
+        "volume" : 0,
+        "weight" : 0,
+        "to_hit" : -4,
+        "rigid": false,
+        "ammo_type" : "223",
+        "capacity" : 200,
+        "count" : 100,
+        "reliability" : 8,
+        "reload_time" : 80,
+        "flags": [ "NO_RELOAD", "NO_UNLOAD" ]
```

Comment:
Too easy to accidentally destroy your ammo belt. I was going to allow disassembly though.